oeps/architectural-decisions/oep-0058-arch-translations-management.rst

```diff
@@ -0,0 +1,262 @@
+placement of the translation files kept in openedx-translations into locally cloned
+repositories for development and containers containing the code translation files are
+formed from. This tool will manage the placement of translation files through an editable
+atlas configuration file (atlas.yml) kept in the repositories that have
+translation files kept in openedx-translations. The atlas.yml file will support
+options that allow for the concatenation, reorganization, and reformatting of translation
+files as they are copied to their locations amongst the code. The atlas.yml file
+will also support selecting which languages to be included in an Open edX deployment. The
+tool will have to be used/ran as part of the setup of a repository, whether for
+development or deployment.
+
+Impacts
+*******
+
+Impact on Translators
+=====================
+
+As we approach the end of the translation upgrade process, we will need to tactically
+move from multiple Transifex projects to a single project. This will require coordination
+with our translators to ensure that moving forward they are providing translations in the
+right place.
+
+Impact on Site Operators
+========================
+
+Currently the translation files for any given service or library is stored at the same
+place as the code, which has generally simplified the deployment story in the past. With
+this change, the translations files will move to their own repository. As we deprecate
+the old translations files, the relevant deployment tooling will need to be updated to
+pull down the translations from the new repository as a part of the deployment process.
+This will impact both the old Ansible based tooling as well as any new docker based
+tooling.
+
+Impact on Developers
+====================
+
+While it won’t directly impact the day-to-day workflow of developers, due to the same
+reasons that we impact site operators (new translations location), we will have to update
+development tools as well. In addition, we will create new instructions for developers on
+how to enable translations for a new service/repo when it comes online.
+
+Locations
+*********
+
+Dumps of the translation/localization files from Transifex for the Open edX Release
+project already exist in a repository with the name of openedx/openedx-i18n. A new
+repository named openedx/openedx-translations will be similarly structured, but it will
+contain the translation files for all repositories within openedx. The
+`Transifex GitHub App`_ will be installed in the openedx organization. Similar to how the
+Build-Test-Release Working Group creates a new branch for each new named release of
+edx-platform, translation releases can also be kept in branches corresponding to
+edx-platform releases.
+
+Rejected Alternatives
+*********************
+
+Rewriting the Current Tooling for the New API
+=============================================
+
+The source code for the edx-transifex-bot can be found in `ecommerce-scripts`_. We could
```

Comment:
This OEP-58 will limit the actions the edx-transifex-bot, but we cannot fully deprecate the bot because it would be instead used as a virtual user that will act as the point of contact in the github action that generates translation source files and the GitHub Transifex App.

Reply:
just to add context here, if you look through https://github.com/openedx/openedx-translations/commits/main you'll see commits by `edx-transifex-bot` starting with `chore: add extracted translation source files`